lib/Frontend/CompilerInvocation.cpp

```diff
@@ -737,7 +738,10 @@ void LangOptions::setCxxInteropFromArgs(ArgList &Args,
     // version, and is either 4, 5, 6, or 7 (even though only 5.9 and 6.* make
     // any sense). For now, we don't actually care about the version, so we'll
     // just use version 6 (i.e., 'swift-6') to mean that C++ interop mode is on.
-    if (EnableCXXInterop)
+    //
+    // FIXME: Always declare the 'Darwin' module as formally having been built
```

Comment:
This `FIXME` reads like this is something should be fixed to do, not what currently do that needs to be fixed. Also there is a separate check for `Darwin` module name that probably also requires a `FIXME`